### ISSUE TITLE
happymh can't pass cloudflare after changing  effective UA

### ISSUE THREAD

**Issue by mojunisme** [tachiyomiorg/tachiyomi-extensions#15713]
### Source information and language

happymh 1.4.2

### Steps to reproduce

1. Open or download a manga chapter directly in the app.
2. failed to pass cloudflare

1. open by webview in app.
2. open a new chapter 
3. check by cloudflare and pass 
4. repeat step 3 for unlimited times.( I can't really pass cloudflare)

Change the UA can't solve this problem, though the UA is from my browser(mobile edge 110.0.1587.66) which can pass cloudflare and open the same manga chapter mentioned before.

### Expected behavior

pass the cloudflare check,  open and download the chapter normally

### Actual behavior

Being blocked by cloudflare, being check umlimitedly, can't really pass the check.
I can't read or download in app and webview.

### Tachiyomi version

0.14.5

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
